Polyglot.Notebook.Docfx.Plugin.Docs/toc.yml

```
@@ -1,3 +1,5 @@
 items:
   - name: About
     href: index.ipynb
+  - name: Getting Started
+    href: getting-started.ipynb
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)